### PR TITLE
Updating metacell normalization

### PR DESCRIPTION
**Pull request by ManuSetty**

### CODE REVIEW

**Comment by ManuSetty** [comment]
@christinedien  This is a pull request to tweak the normalization of metacells in preparation step. Please verify the steps for integration data before accepting the pull request

File: SEACells/genescores.py [hunk 1]
```diff
@@ -50,130 +34,157 @@ def prepare_multiome_anndata(atac_ad, rna_ad, SEACell_label='SEACell', n_bins_fo
...
-    svd = pd.DataFrame(atac_mod_ad.obsm['X_svd'], index=atac_mod_ad.obs_names)
-    summ_svd = svd.groupby(atac_mod_ad.obs[SEACell_label]).mean()
-    atac_meta_ad.obsm['X_svd'] = summ_svd.loc[atac_meta_ad.obs['original_atac'], :].values
+    # ATAC - Normalize
+    sc.pp.filter_genes(atac_meta_ad, min_cells=1)
+    _normalize_ad(atac_meta_ad)
+    _add_atac_meta_data(atac_meta_ad)
     
     # remove peaks with zero counts
     sc.pp.filter_genes(atac_meta_ad, min_cells=1)
```
Comment (christinedien) [inline SEACells/genescores.py]: Lines 148-150 can be removed

File: SEACells/genescores.py [hunk 2]
```diff
@@ -50,130 +34,157 @@ def prepare_multiome_anndata(atac_ad, rna_ad, SEACell_label='SEACell', n_bins_fo
...
     atac_meta_ad.obs['original_atac'] = mapping['atac'].values
     
-    # Add summarized X_svd representation
-    svd = pd.DataFrame(atac_mod_ad.obsm['X_svd'], index=atac_mod_ad.obs_names)
-    summ_svd = svd.groupby(atac_mod_ad.obs[SEACell_label]).mean()
-    atac_meta_ad.obsm['X_svd'] = summ_svd.loc[atac_meta_ad.obs['original_atac'], :].values
+    # ATAC - Normalize
+    sc.pp.filter_genes(atac_meta_ad, min_cells=1)
+    _normalize_ad(atac_meta_ad)
+    _add_atac_meta_data(atac_meta_ad)
```
Comment (christinedien) [inline SEACells/genescores.py]: Just need to pass through params:
`_add_atac_meta_data(atac_meta_ad, atac_mod_ad, n_bins_for_gc)`

**Review by christinedien — CHANGES_REQUESTED** [review]
Hi Manu,
Just two things to fix for the integrate methods:

- [x] Passing parameters to `_add_atac_meta_data` function
- [x] Removing the duplicate filtering/ normalization in lines 148-150

**Review by christinedien — APPROVED** [review]

**Pull request merged**